recipients_app/lib/view/pages/payments_page.dart

```diff
@@ -7,26 +7,34 @@ import "package:flutter/material.dart";
 import "package:flutter_bloc/flutter_bloc.dart";
 import "package:flutter_gen/gen_l10n/app_localizations.dart";
 
-class PaymentsPage extends StatelessWidget {
+class PaymentsPage extends StatefulWidget {
   const PaymentsPage({super.key});
 
+  @override
+  State<PaymentsPage> createState() => _PaymentsPageState();
+}
+
+class _PaymentsPageState extends State<PaymentsPage> {
+  final _refreshIndicatorKey = GlobalKey<RefreshIndicatorState>();
+
   @override
   Widget build(BuildContext context) {
     final localizations = AppLocalizations.of(context)!;
 
     final recipient = context.watch<AuthCubit>().state.recipient;
-    final paymentsUiState =
-        context.watch<PaymentsCubit>().state.paymentsUiState!;
+    final paymentsUiState = context.watch<PaymentsCubit>().state.paymentsUiState!;
 
     return Scaffold(
       appBar: AppBar(
         elevation: 0,
         title: Text(localizations.payments),
         centerTitle: true,
       ),
-      body: Padding(
-        padding: AppSpacings.h8,
-        child: Column(
+      body: RefreshIndicator(
```

Comment:
Something seems broken with scrolling. Like the nested listview is not scrolling at all. I can only scroll on the header 🤔